lib/common/schemas.c

```diff
@@ -1053,6 +1053,21 @@ get_schema_version(const char *name)
 }
 
 /* set which validation to use */
+/*!
+ * \brief Update CIB XML to latest schema that validates it
+ *
+ * \param[in,out] xml_blob   XML to update (may be freed and replaced after
+ *                           being transformed)
+ * \param[out]    best       If not NULL, set to schema index of latest schema
+ *                           that validates \p xml_blob
+ * \param[in]     max        If positive, do not update \p xml_blob to any
+ *                           schema past this index
+ * \param[in]     transform  If false, do not update \p xml_blob to any schema
+ *                           that requires an XSL transform
+ * \param[in]     to_logs    Has something to do with logging
```

Comment:
It looks like if this is true, parser errors and warnings will be logged and if it's false, they'll be sent to stderr via fprintf.

Reply:
Yeah for the most part, but there seems to be some other behavior I'm not sure about. I'll put that though :)

Reply:
Eventually we'll get rid of all that and use `pcmk__output_t` anyway